### PR TITLE
adds tyr helmet attachment

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
makes tyr mod 1 come in a set
+10 armor

## Why It's Good For The Game
tyr helmet attachment has been needed for a while

## Changelog
:cl:
add: Added new tyr helmet attachie. +10 To all armor stats. 15 to melee.
/:cl:

### CODE REVIEW

**Comment by turnop** [comment]
yes god please

**Review by Kuro020 — APPROVED** [review]
sprite good

**Comment by Tristan63** [comment]
ballistic helmet module incoming?

**Review by TiviPlus — APPROVED** [review]

**Comment by bowlofcereal** [comment]
This existed a long while ago im pretty sure I dont remember why it was removed but not having it invalidates the tyr mk2 because of head aiming meta

**Review by SplinterGP — CHANGES_REQUESTED** [review]
add it for tyr mk2

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -899,6 +899,13 @@
 		/obj/item/paper/chemsystem,
 	)
 
+/obj/effect/essentials_set/tyr
+	desc = "A set of specialized gear for improved close-quarters combat longevitiy."
+	spawned_gear_list = list(
+		/obj/item/helmet_module/attachable/tyr_head,
+		/obj/effect/essentials_set/tyr,
```
Comment (SplinterGP) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:906]: Think thats a typo

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**